src/screens/WhotoFollow/index.js

```diff
@@ -160,9 +164,15 @@ const WhotoFollow = () => {
         const uploadedImageUrl = await ImageUtils.uploadImageWithoutAuth(
           data?.users?.profile_pic_path
         );
+        AnalyticsEventTracking.eventTrack(
+          BetterSocialEventTracking.ONBOARDING_REGISTRATION_UPLOAD_IMAGE_SUCCESS
+        );
         data.users.profile_pic_path = uploadedImageUrl?.data?.url;
         console.log('uploadedImageUrl', uploadedImageUrl);
       } catch (e) {
+        AnalyticsEventTracking.eventTrack(
+          BetterSocialEventTracking.ONBOARDING_REGISTRATION_UPLOAD_IMAGE_FAIL
+        );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of analytics event tracking for image upload success and failure is a good practice for monitoring user interactions. However, the error object `e` should be logged or sent to your error tracking system in the catch block for better debugging.
```diff
- console.log('error upload', e);
+ console.log('error upload', e);
+ AnalyticsEventTracking.eventTrack(
+   BetterSocialEventTracking.ONBOARDING_REGISTRATION_UPLOAD_IMAGE_FAIL,
+   { error: e.message }
+ );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->